nb2/05.02-V2-Introducing-Scikit-Learn.ipynb

```diff
@@ -2,10 +2,7 @@
  "cells": [
```

Comment:
<p>This is <s>striked fsfs ffsd</s> and this is <strong>bold text</strong> </p><p> </p><ul><li>A</li><li>B</li><li>C</li><li>D</li></ul><p> </p><ol><li>One</li><li>Two</li><li>Three</li></ol><p> </p><p> </p><h2>Header 2</h2><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>